transit_odp/timetables/extract.py

```diff
@@ -137,14 +139,52 @@ def extract(self) -> ExtractedData:
             columns=["file_id", "route_link_ref", "from_stop_ref", "to_stop_ref"]
         ).set_index(["file_id", "route_link_ref"])
 
+        # extract flexible journey patterns
+        logger.debug("Extracting flexible journey patterns")
+        flexible_journey_patterns = self.extract_flexible_journey_patterns()
+        logger.debug("Finished extracting flexible journey patterns")
+
+        logger.debug("Extracting flexible journey pattern sections")
+        flexible_jp_sections, flexible_jp_to_jps = self.create_flexible_jps(
+            flexible_journey_patterns
+        )
+        logger.debug("Finished extracting flexible journey pattern sections")
+
+        logger.debug("Extracting flexible journey pattern timing links")
+
+        flexible_timing_links = self.create_flexible_timing_link(
```

Comment:
The function seems to be suited for transformation stage

Reply:
same